cargo-concordium/src/build.rs

```diff
@@ -438,7 +453,17 @@ pub fn build_and_run_wasm_test(extra_args: &[String]) -> anyhow::Result<bool> {
                     "    {} ... {}",
                     Color::Red.bold().paint("Error"),
                     Style::new().italic().paint(err.to_string())
-                )
+                );
+                if let utils::ReportError::Reported {
+                    quickcheck: true, ..
+                } = err
+                {
+                    eprintln!(
+                        "    {}: {}",
+                        Style::new().bold().paint("Seed"),
+                        Style::new().bold().paint(seed_u64.to_string())
+                    )
+                };
```

Comment:
With respect to the comments in base.

If you want to keep this flag in the `Reported` struct, as opposed to looking into the test-host then I would recommend renaming the `quickcheck` flag to something more descriptive. For example `randomness_used` is more appropriate.

Reply:
Done, return a pair `(ReportError, bool)` instead and remove the `quickcheck` flag.